conf/SoftwareProviders.json

```diff
@@ -4175,6 +4175,21 @@
     "hearing": false,
     "motor": false
   },
+  {
+    "name": "SAP Document and Reporting Compliance",
```

Comment:
did you check with Fransis about existing name (SAP Document and Reporting Compliance Service ). Both companies are different then we are good.

Reply:
yes, i have. they're different

Reply:
Good. This kind of conersations, please put it in the ticket going forward. Thanks